pkg/client/unversioned/client.go

```diff
@@ -34,6 +34,7 @@ import (
 // an interface to allow mock testing.
 type Interface interface {
 	PodsNamespacer
+	PodLogsNamespacer
```

Comment:
We follow this pattern of branching out log subresources in Openshift and it feels much cleaner than bloating existing clients.